OoyalaSkinSampleApp/app/src/main/java/com/ooyala/sample/lists/MultiAudioListActivity.java

```diff
@@ -0,0 +1,95 @@
+package com.ooyala.sample.lists;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.AdapterView.OnItemClickListener;
+import android.widget.ArrayAdapter;
+import android.widget.ListView;
+
+import com.ooyala.sample.R;
+import com.ooyala.sample.players.MultiAudioActivity;
+import com.ooyala.sample.utils.PlayerSelectionOption;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class MultiAudioListActivity extends Activity implements OnItemClickListener {
+
+  public final static String getName() {
+    return MultiAudioListActivity.class.getSimpleName();
+  }
+
+  private static Map<String, PlayerSelectionOption> selectionMap;
+  private ArrayAdapter<String> selectionAdapter;
+
+  /**
+   * Called when the activity is first created.
+   */
+  @Override
+  public void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    setTitle(getName());
+
+    String pcode = "k0a2gyOt0QGNJLSuzKfdY4R-hw2b";
+    String domain = "http://www.ooyala.com";
+
+    selectionMap = new LinkedHashMap<>();
+
+    selectionMap.put("MT-1 ENG/GER", new PlayerSelectionOption("wxNjRwZTE6bxMBkrQbfM_BwokGf0pyOm", pcode, domain, MultiAudioActivity.class));
+    selectionMap.put("MT-2 UND/UND", new PlayerSelectionOption("xmaTRwZTE6lCuPd3H82AteBFUvNHtrHu", pcode, domain, MultiAudioActivity.class));
+    selectionMap.put("MT-3 UND/UND", new PlayerSelectionOption("Y1cTRwZTE6Mc47fc_8n161HVYQYu5l0d", pcode, domain, MultiAudioActivity.class));
+    selectionMap.put("MT-4 ENG(main)/ENG(comm)/GER(main)/GER(comm)+no config", new PlayerSelectionOption("txcjRwZTE6aMSqloxZZ3hZhTqz5hEaY9", pcode, domain, MultiAudioActivity.class));
+    selectionMap.put("MT-5 ENG(main)/ENG(comm)/GER(main)/GER(comm)+config", new PlayerSelectionOption("EweDRwZTE6ipjPDynkEotrpTvvYgDvr7", pcode, domain, MultiAudioActivity.class));
+    selectionMap.put("MT-6 Mixed with Language code and UND", new PlayerSelectionOption("o3eTRwZTE6TBJnp6gPJia25dtOcrugUk", pcode, domain, MultiAudioActivity.class));
+
+//    selectionMap.put("HLS 1 AAC 2 tracks", new PlayerSelectionOption("Rwc3luZTE6JAVo4Tb70Ioc-9goU1fswD", pcode, domain, MultiAudioActivity.class));
```

Comment:
Do we need these tracks commented? Why don't we uncomment them or just remove them?